app/views/partners/helps/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<section class="content">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        <!-- Default box -->
+        <div class="card card-info">
+          <div class="card-header">
+            <h2 class="card-title">Frequently Asked Questions</h2>
+          </div>
+          <div class="card-body p-0">
+            <ul id="faq-list">
+              <li>How do I edit my organization?</li>
+              <ul id="faq-list">
```

Comment:
Same as comment for banks: won't want to have same ID for more than one element in the page.